.github/workflows/build_test_publish_release.yml

```diff
@@ -32,22 +32,10 @@ jobs:
           set -e # abort on error
           set -x # show commands
 
-          set -o allexport
-          source ./variables.env
-
           git config --global user.name 'wikibase suite github actions bot'
           git config --global user.email 'wikibase-suite-github-actions-bot@users.noreply.github.com'
 
-          tag_name="v${VERSION}"
-
-          if git tag "$tag_name"; then
-            git push --tags origin "$tag_name"
-          else
-            echo "*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***"
-            echo "Cannot tag ${tag_name}. Most probably this tag is already given to another commit."
-            echo "Make sure to update the VERSION variable in variables.env to publish a new deploy release."
-            echo "*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***"
-          fi
+          ./nx release --dry-run
```

Comment:
Before merge, I guess this should not be a dry run anymore.

Reply:
Now that things are setup for manually triggering the releases the Dry Run is now an interactive Publish workflow option (with a default of true)